recipe/meta.yaml

```diff
@@ -1,61 +1,284 @@
+    sha256: 37cd629a75d2a34974a3d771ccab5c9e3caf3d138cea81040f9cdba1aded1cf2         # [win]
+  - url: https://github.com/python/cpython-source-deps/archive/tix-8.4.3.6.zip       # [win]
+    folder: externals/tix-8.4.3.6                                                    # [win]
+    sha256: e558e3dc5e67ac0942f8fceafce00ca46b177da9ebeaf38ec7fafd9b9913ac56         # [win]
+  - url: https://github.com/python/cpython-source-deps/archive/bzip2-1.0.6.zip       # [win]
+    folder: externals/bzip2-1.0.6                                                    # [win]
+    sha256: c42fd1432a2667b964a74bc423bb7485059c4a6d5dc92946d59dbf9a6bdb988d         # [win]
+  - url: https://github.com/python/cpython-source-deps/archive/zlib-1.2.11.zip       # [win]
+    folder: externals/zlib-1.2.11                                                    # [win]
+    sha256: debb1952945fa6c25817a40abe90641b572c83171f244937b70b9fe156f5a63a         # [win]
+  - url: https://github.com/python/cpython-bin-deps/archive/nasm-2.11.06.zip         # [win]
+    folder: externals/nasm-2.11.06                                                   # [win]
+    sha256: de3c87b26a80e789986d8e6950c6304175d3829afe9c6c7211eb7257266ab0ac         # [win]
+  - url: https://github.com/python/cpython-bin-deps/archive/libffi.zip               # [win]
+    folder: externals/libffi                                                         # [win]
+    sha256: 4872e72e188a5aa1124db0c3b163a4163e84ead359a514d86dd7c6fa2d2ff02a         # [win]
 
 build:
-  number: 0
-  # Windows has issues updating python if conda is using files itself.
-  # Copy rather than link.
-  no_link:
-    - DLLs/_ctypes.py   # [win]
-    - bin/python3.6     # [unix]
-  track_features:
-    - vc14              # [win]
-
-requirements:
-  build:
-    - bzip2 1.0.*       # [unix]
-    - openssl 1.0.*     # [unix]
-    - readline 6.2*     # [unix]
-    - sqlite 3.13.*     # [unix]
-    - tk 8.5.*          # [unix]
-    - xz 5.2.*          # [unix]
-    - zlib 1.2.*        # [unix]
-    - ncurses 5.9*      # [linux]
-  run:
-    - openssl 1.0.*     # [unix]
-    - readline 6.2*     # [unix]
-    - sqlite 3.13.*     # [unix]
-    - tk 8.5.*          # [unix]
-    - xz 5.2.*          # [unix]
-    - zlib 1.2.*        # [unix]
-    - ncurses 5.9*      # [linux]
-    - vs2015_runtime    # [win]
-
-test:
-  commands:
-    - python -V
-    - python3 -V            # [unix]
-    - 2to3 -h
-    - pydoc -h
-    - python3-config --help  # [unix]
+  number: 8
+  script_env:
+    - PY_INTERP_LINKAGE_NATURE
+    - PY_INTERP_DEBUG
+
+outputs:
+  - name: python
+    build:
+      activate_in_script: True
+      # Windows has issues updating python if conda is using files itself.
+      # Copy rather than link.
+      no_link:
+        - DLLs/_ctypes.pyd  # [win]
+      # We delete the shared libraries.
+      ignore_run_exports:   # [unix]
+        - bzip2             # [unix]
+      # Disabled until new conda-build release
```

Comment:
@mingwandroid and @isuruf do you know what is the status of this? Can we just remove the commented out lines below?